### PR TITLE
Add validate method

### PR DESCRIPTION
**Pull request by radekzawada**

### CODE REVIEW

File: lib/operations.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@
 require "active_support/core_ext/module/delegation"
 require "active_support/inflector/inflections"
 require "active_model/naming"
-require "active_model/errors"
+require "active_model/errors" if RUBY_VERSION.to_d < 3.1
```
Comment (pyromaniac) [inline lib/operations.rb:12]: Why did you add this? This doesn't solve the problem? I would keep 3.1 failing for a while

Comment (pyromaniac) [reply lib/operations.rb:12]: Also, Gem::Version

Comment (pyromaniac) [reply lib/operations.rb:12]: But but then how specs are green then?

**Review by pyromaniac — APPROVED** [review]

File: lib/operations.rb (hunk 1 above)
Comment (radekzawada) [reply lib/operations.rb:12]: apparently ruby 3.1 does not need that :D

Comment (pyromaniac) [reply lib/operations.rb:12]: but it can't be true :( the gem needs it

Comment (pyromaniac) [reply lib/operations.rb:12]: Actually, maybe no indeed, can you drop the line completely?

**Pull request merged**